src/Provider/Provider.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -99,43 +99,40 @@ private int SanitizePage(int page, int totalVacancies)
             return null;
         }
 
-        private List<SelectListItem> GetFilterSelectOptions(List<VacancySummary> vacancies, VacancyStatus? filterStatus)
+        private List<SelectListItem> GetFilterSelectOptions(VacancyStatus? filterStatus)
         {
             return new List<SelectListItem>
             {
-                GetFilterSelectListItem(vacancies, null, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Draft, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Submitted, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Live, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Closed, filterStatus),
-                GetFilterSelectListItem(vacancies, VacancyStatus.Referred, filterStatus),
+                GetFilterSelectListItem(null, filterStatus),
+                GetFilterSelectListItem(VacancyStatus.Live, filterStatus),
+                GetFilterSelectListItem(VacancyStatus.Referred, filterStatus),
```

Comment:
Why do we have a SelectListItem in the orchestrator, that is a View concern

Reply:
This is build on the existing filter functionality done by Chris. I have only modified the text as per the ticket.

Reply:
@sachpatel Orchestrators are ViewModel aware I used `SelectListItem` as I didn't see the point in create our own VM to do this.

Reply:
ViewModel aware yes but we haven't been mixing MVC into Orchestrators previously or HTTP 😬 . Having a look this is going to be quite a bit to untangle so I'll leave it for you guys to manage with the changes being made to the dashboard.

Reply:
@sachpatel  No untangling to be done. We can just create our own class with `Text`, `Value` & `Selected` properties.

Thinking about it - the drop down is now completely static so we don't need to do any of this anymore we can just put it in the view.  We had to construct the list server-side previously so we could include the counts.  @shomavg Can you kill all this and just add the <options> in the HTML?